src/rules/at-root-no-redundant-selector/README.md

```diff
@@ -0,0 +1,67 @@
+# at-root-no-redundant-selector
```

Comment:
I had initially suggested `no-redundant-at-root` as well, @nex3 suggested `at-root-no-redundant-selector`, thoughts? 🤔

Reply:
Maybe we can discuss this today at coworking time, I still think that `no-redundant-at-root` is a more accurate name, since the selector is not the problem, even less so when the `@at-root` is already at the root

Reply:
discussed this briefly, the id should indeed start with `at-root` following the same pattern as other checks, so I think then the ID should be `at-root-no-redundant`, without the "selector" part 

ultimately the final say will be from the maintainer :)